doc/maths.md

```diff
@@ -231,9 +231,10 @@ For more than `4` dimensions, you can use a storage as argument: `y = torch.ones
 
 For more than 4 dimensions, you can use a storage as argument: `y = torch.rand(torch.LongStorage{m, n, k, l, o})`.
 
+`y = torch.rand(gen, m, n)` returns a `m × n` `Tensor` of random numbers from a uniform distribution on the interval `[0, 1)`, using the non-global random number generator `gen` created by [Generator()](random.md#torch.Generator).
```

Comment:
A few tweaks:

"the non-global" -> "a non-global"

"[Generator()]" -> "[`torch.Generator()`]"